app/views/images/_index.html.erb

```diff
@@ -83,6 +83,8 @@
             %>
           <% end %>
           <% if warpable.placed? %>
+            <a rel="tooltip" id="locate-warpable-<%= warpable.id %>" title="Pan to this image" style="cursor: pointer" class="btn btn-sm btn-success" style="word-wrap: break-word"><i class='fa fa-crosshairs'></i></a>
+            <a rel="tooltip" title="Download full resolution distorted image (please allow popups)" style="cursor: pointer" class="btn btn-sm btn-outline-secondary img-download img-download-<%= warpable.id %>" data-image="/images/fetch/?url=<%= warpable.image.url(:original) %>"style="word-wrap: break-word"><i class="fa fa-download"></i></a>
```

Comment:
I think, the original issue intend to change the cursor to pointer on existing button. You have to make changes to existing links.

Reply:
@gauravano this PR does change to a pointer on hovering over the buttons(screenshots attached in issue).
Sorry I'm not entirely clear on what you mean. Could you tell me what you mean by 'making changes to existing links'?

Thanks!

Reply:
The links are already present so you just have to make changes. See the next two lines where you added new lines and compare. Aren't they same??